hawtio-system/src/main/java/io/hawt/web/GitFileUploadFilter.java

```diff
@@ -0,0 +1,40 @@
+package io.hawt.web;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GitFileUploadFilter {
```

Comment:
Why do we need a whole new class just to configure it with `GIT_CONFIG`?  If there's any awkwardness that requires it then I guess `GlobalFileUploadFilter` should be refactored to make it more flexible.

Reply:
Well we could do that but the idea is to allow process wide filter configuration as well as the `GlobalFileUploadFilter`, in this case `GitFileUploadFilter`.

Reply:
It doesn't make sense to me. In object-oriented programming, you don't need a class to have different configurations for different processes; you just need different instances from a single class. In this case, `GlobalFileUploadFilter`'s `filters` is an instance field so there's no reason you cannot use it for multiple processes with the different instances of it.

`GIT_CONFIG` can be located where a `GlobalFileUploadFilter` instance for `GitServlet` is created, which is `GitServlet` itself.

Reply:
Fixed in #2388